modules/ossm-performance-scalability.adoc

```diff
@@ -0,0 +1,79 @@
+////
+This module included in the following assemblies:
+- /v2x/customizing-installation-ossm.adoc
+////
+
+[id="ossm-performance-scalability_{context}"]
+= Performance and scalability
+
+{ProductName} strives to provide features with minimal resource overhead and aims to support very large meshes with high request rates while adding minimal latency.
+
+== Performance summary for Istio 1.6.8
+
+The upstream Istio community load tests mesh consists of *1000* services and *2000* sidecars with 70,000 mesh-wide requests per second.
+Running the tests using Istio 1.6.8, generated the following results:
+
+* The Envoy proxy uses *0.5 vCPU* and *50 MB memory* per 1000 requests per second going through the proxy.
+* Istiod uses *1 vCPU* and *1.5 GB* of memory.
+* The Envoy proxy adds *3.12 ms* to the 90th percentile latency.
+* The legacy `istio-telemetry` service (disabled by default in Service Mesh 2.0) uses *0.6 vCPU* per 1000 mesh-wide requests per second for deployments that use Mixer.
+// TODO The Envoy numbers goes down in 1.9, check for the latest data with next version of Istio.
+The data plane components, the Envoy proxies, handle data flowing through the system. The control plane component, Istiod, configures the data plane. The data plane and control plane have distinct performance concerns.
+
+== Control plane performance
+
+Istiod configures sidecar proxies based on user authored configuration files and the current state of the system.
+In a Kubernetes environment, Custom Resource Definitions (CRDs) and deployments constitute the configuration and state of the system.
+The Istio configuration objects like gateways and virtual services, provide the user-authored configuration.
+To produce the configuration for the proxies, Istiod processes the combined configuration and system state from the Kubernetes environment and the user-authored configuration.
+
+The control plane supports thousands of services, spread across thousands of pods with a similar number of user authored virtual services and other configuration objects.
+Istiod's CPU and memory requirements scale with the number of configurations and possible system states.
+The CPU consumption scales with the following factors:
+
+* The rate of deployment changes.
+* The rate of configuration changes.
+* The number of proxies connecting to Istiod.
+
+however this part is inherently horizontally scalable.
```

Comment:
s/however/However. Or maybe this is part of the last bullet point, I'm not sure.